onboarding/chat-moderators.md

```diff
@@ -28,6 +28,9 @@ Removal of the role is subject to the discretion of the Core team members and ma
 ### Maintain, follow, and enforce rules
 
 - As a moderator, you will be expected to **adhere to** and **uphold** the rules as detailed and maintained in the `#rules` channel in the Discord at all times. 
+- **Following the matrix**
+    - When enforcing rules **following the matrix is NOT optional** and should be used to determine when a user should be banned and enforcement of rules should be done in line with the matrix.
+    - If you notice a potential rule break it's important to consider the context and ensure that when enforcement occurs it is in line with the [matrix](https://github.com/TheOdinProject/top-meta/blob/main/MODERATION/Moderation-Matrix.md) and uses the zap system from the matrix.
```

Comment:
The only thing I think *needs* to be changed is the "Following the matrix" heading to what is in the suggestion below. The rest of the suggestion is kind of take or leave and more just throwing out there other ideas in terms of verbiage.

The second sub-bullet here I wasn't really too sure about, but wanted to change it based on my suggested change to the first sub-bullet.

```suggestion
- **Follow the [moderation matrix](https://github.com/TheOdinProject/top-meta/blob/main/MODERATION/Moderation-Matrix.md)**
    - When enforcing the rules, **following the moderation matrix is NOT optional**. The moderation matrix must be used to determine what zap level a user should receive for a particular rule break, and zap levels should not be skipped under any circumstances.
    - If you notice a potential rule break it can be important to consider the context. What might look like an unsolicited ping might be innocent rapport between two users who have a history with one another.
```